swim/memberlist_test.go

```diff
@@ -201,9 +201,12 @@ func (s *MemberlistTestSuite) TestMultipleUpdates() {
 }
 
 func (s *MemberlistTestSuite) TestUpdateTriggersReincarnation() {
+	source := "192.0.2.1:1234"
+	s.NotEqual(source, s.m.local.Address, "this test relies on the source and the target of the change to be different")
```

Comment:
addresses are not validated, so we can even use simple words. From what I understand happening here is that it does only create the swin state machine (`memberlist`) in memory and executes updates against it. But other tests also use ip addresses here. Although `127.0.0.1:xxxx` in this case, the reason to use `192.0.2.x:xxxx` here is that we settled on using TEST-NET-1 as specified in [rfc5735](https://tools.ietf.org/html/rfc5735) for our tests going forward.